bin/stop-service.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+DIRNAME=`dirname $0`
+cd $DIRNAME/..
+
+readonly ARREBOL_PORT=8080
+
```

Comment:
is it the usual way to stop things in the context of springboot apps? I think lsof'ing things is not very reliable (at least, not much readable). Maybe, using pgrep or writing the pid of the process after calling the start script